BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Requests/RegisterRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BackendApartmentReservation.DataContracts.DataTransferObjects.Requests
+{
+    public class RegisterRequest
+    {
+        [Required]
```

Comment:
Add min length for both name and surname

Reply:
Is it needed? Names can be 1 letter long https://www.quora.com/Can-you-have-a-single-letter-as-a-legal-first-name

Reply:
Forgot to add min length of 1 :D so you do not post empty string